pkg/settings/setting.go

```diff
@@ -16,7 +16,17 @@ import (
+const (
+	RancherVersionDev = "2.7.99"
+
+	// ClusterAgentAffinity used to be hardcoded in the agent deployment template but is now defined here as the
+	// single source of truth.
+	ClusterAgentAffinity = "{\n  \"nodeAffinity\": {\n    \"requiredDuringSchedulingIgnoredDuringExecution\": {\n      \"nodeSelectorTerms\": [\n        {\n          \"matchExpressions\": [\n            {\n              \"key\": \"beta.kubernetes.io/os\",\n              \"operator\": \"NotIn\",\n              \"values\": [\n                \"windows\"\n              ]\n            }\n          ]\n        }\n      ]\n    },\n    \"preferredDuringSchedulingIgnoredDuringExecution\": [\n      {\n        \"weight\": 100,\n        \"preference\": {\n          \"matchExpressions\": [\n            {\n              \"key\": \"node-role.kubernetes.io/controlplane\",\n              \"operator\": \"In\",\n              \"values\": [\n                \"true\"\n              ]\n            }\n          ]\n        }\n      },\n      {\n        \"weight\": 100,\n        \"preference\": {\n          \"matchExpressions\": [\n            {\n              \"key\": \"node-role.kubernetes.io/control-plane\",\n              \"operator\": \"In\",\n              \"values\": [\n                \"true\"\n              ]\n            }\n          ]\n        }\n      },\n      {\n        \"weight\": 100,\n        \"preference\": {\n          \"matchExpressions\": [\n            {\n              \"key\": \"node-role.kubernetes.io/master\",\n              \"operator\": \"In\",\n              \"values\": [\n                \"true\"\n              ]\n            }\n          ]\n        }\n      },\n      {\n        \"weight\": 1,\n        \"preference\": {\n          \"matchExpressions\": [\n            {\n              \"key\": \"cattle.io/cluster-agent\",\n              \"operator\": \"In\",\n              \"values\": [\n                \"true\"\n              ]\n            }\n          ]\n        }\n      }\n    ]\n  },\n  \"podAntiAffinity\": {\n    \"preferredDuringSchedulingIgnoredDuringExecution\": [\n      {\n        \"weight\": 100,\n        \"podAffinityTerm\": {\n          \"labelSelector\": {\n            \"matchExpressions\": [\n              {\n                \"key\": \"app\",\n                \"operator\": \"In\",\n                \"values\": [\n                  \"cattle-cluster-agent\"\n                ]\n              }\n            ]\n          },\n          \"topologyKey\": \"kubernetes.io/hostname\"\n        }\n      }\n    ]\n  }\n}"
+
+	// FleetAgentAffinity is hardcoded in the rancher/fleet repo in the agent manifest
+	// https://github.com/rancher/fleet/blob/90e33140906ba5d4931b4e1dee588854cbb300b1/pkg/agent/manifest.go#L178-L195
+	FleetAgentAffinity = "{\"nodeAffinity\":{\"preferredDuringSchedulingIgnoredDuringExecution\":[{\"weight\":1,\"preference\":{\"matchExpressions\":[{\"key\":\"fleet.cattle.io/agent\",\"operator\":\"In\",\"values\":[\"true\"]}]}}]}}"
+)
```

Comment:
Is there any particular reason why this isn't defined via a raw string literal, i.e. 
```
ClusterAgentAffinity = `{
  "nodeAffinity": {
# etc.
}
`
```